client/src/app/admin/admin.component.html

```diff
@@ -12,10 +12,10 @@
       </div>
       <div class="nav__side-nav__navbar-header__profile-container">
         <div class="nav__side-nav__navbar-header__profile-container-name">
-          {{user?.firstName}} {{user?.lastName}}
+          {{(userService.$user | async)?.firstName}} {{(userService.$user | async)?.lastName}}
```

Comment:
@mohsinayaz instead of using `userService` directly into the html, create an observable `$user` in the component and use that. Same as we're doing at other places.